src/features/ui/FormInputs/FormInputs.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC, useState } from 'react';
+
+import { Button, Input } from '@zero-tech/zui/components';
+
+import styles from './FormInputs.module.scss';
+
+export interface FormInputsProps {
+	action: 'transfer';
+	label?: string;
+	isError?: boolean;
+	errorMessage?: string;
```

Comment:
This looks like duplicated logic - `errorMessage !== undefined` is probably equivalent to `isError === true`. You should be able to remove `isError`, and just use `errorMessage !== undefined`.

Reply:
Sweet thanks, updated!